### ISSUE TITLE
DHW

### ISSUE THREAD

**Issue by glynhudson** [openenergymonitor/heatpumpmonitor.org#12]
It would be a good idea to record display which systems are doing DHW + CH and which systems are just doing CH. Eg Adams SCOP is very impressive but obviously needs to be caveated and can't be directly compared with a system that's doing CH + DHW

**Comment by davidMbrooke** [comment]
A few thoughts on how this might be implemented:

- A simple boolean indicator of Yes or No for DHW heating
- Some sort of _qualitative_ indicator of _how much_ DHW is being delivered by the heat pump, for which candidates include:
  - Whether there are complementary sources of DHW (e.g. solar thermal or solar PV divert)?
  - How many occupants are consuming DHW?
  - To what temperature is the stored DHW being heated?
- Some sort of _quantitative_ indicator of _how much_ DHW is being delivered by the heat pump, for which candidates include:
  - How many kWh of heat does the heat pump send to DHW?
  - What proportion of the time does the heat pump spend on DHW versus CH?
  - What quantity of DHW is being consumed by the household (litres per day if that is metered)?

**Comment by zarch1972** [comment]
Is this issue for adding the ability to monitor DHW runs separately from heating?  Or do we need another issue starting?
We'd spoken about using CT clamp around DHW pump to differentiate between heating and hot water generation.

emoncms then picking that up and being able to show COP for each of heating and DHW for those that can CT clamp the DHW times?

**Comment by davidMbrooke** [comment]
I'm treating this Issue as what we'd change in _heatpumpmonitor.org_ to have that provide a summary-level view of which systems listed there are CH-only versus mixed DHW and CH (and then ideally some indication of _how_ mixed each one is).

I agree with you Mick that it would be great to have some metrics that give us heat-to-CH versus heat-to-DHW and the ability to measure CoP for CH versus CoP for DHW. I presume that means changing the MyHeatpump app (as distinct from heatpumpmonitor.org) so would the better place for an issue on that subject by the 'app' repo https://github.com/emoncms/app/issues ?

(While that would probably be the strictly-correct thing to do, the distinction might be lost on most people, so maybe we keep all the issues together, in here?)

So, in summary:
* I think we do need another Issue that lets us monitor DHW runs separately
* I'll defer to others to decide which GitHub repo that other issue should be raised in

**Comment by MrTimbones** [comment]
> It would be a good idea to record display which systems are doing DHW + CH and which systems are just doing CH

> A simple boolean indicator of Yes or No for DHW heating

The site already captures which systems do or do not have have DHW in the `DHW control type` field.

When we're able to do advanced filtering (#10) then it'll be possible to exclude systems without DHW, for example.

> * Whether there are complementary sources of DHW (e.g. solar thermal or solar PV divert)?
> * How many occupants are consuming DHW?
> * To what temperature is the stored DHW being heated?

This last item is captured in the system form, but the other two are not. Should we add them? Or is the form long enough?

> Is this issue for adding the ability to monitor DHW runs separately from heating?

This is now provided directly by My Heatpump app, though HeatpumpMonitor doesn't aggregate this data yet -> #41
